doc/circuit_assert_is_zero.md

```diff
@@ -0,0 +1,36 @@
+# quantestpy.circuit.assert_is_zero
+
+## circuit.assert_is_zero(circuit, qubits=None, atol=1e-8, msg=None)
+
+Raises a QuantestPyAssertionError if qubits of the circuit are either not 0 or entangled with other qubits up to desired tolerance.
+
+The circuit `circuit` is converted to an operator and eventually to a state vector. The test verifies
+```py
+abs(selected elements of the state vector) <= atol
+```
+where `selected elements of the state vector` can be anticipated from `qubits`.
```

Comment:
I added an explanation and modified it as

> It can be shown that, when some qubits are 0 and not entangled with other qubit(s), certain elements of the state vector are also 0. This is verified by the assert method, namely
> ```py
> abs(certain element(s) of the state_vector) <= atol
> ```
> where `certain element(s)` are uniquely determined from `qubits`.

Is it now clear?

Reply:
I left my comment above.